### PR TITLE
prepare for next round of releases

### PR DESCRIPTION
**Pull request by acao**
- add changelog
- remove peer deps
- minor version bumps for GLS + codemirror because of license change
- everything ready for next release but GraphiQL

### CODE REVIEW

**Comment by acao** [comment]
@lostplan still trying to get you team access, haha. I'm pretty sure i captured everything in the changelog?

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+## Codemirror GraphQL -  0.9 - 11 Aug, 2019
```
Comment (benjie) [inline CHANGELOG.md]: Can we add an introductory header above this?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+- Add unit test coverage
+- Switch to Jest
+
+
+## Codemirror Graphql Mode 0.8.4 - 11 Aug, 2018
+You will now be importing async methods from gls-interface 2.0.0, thus your bundler will require regenerator runtime
+
+## Chores
+- BREAKING - Use GLS interface/parser 2.0.0 for graphql 14
+- BREAKING - This introduces
```
Comment (benjie) [inline CHANGELOG.md]: This introduces...?

File: resources/prepublish.js [hunk 3]
```diff
@@ -34,24 +34,24 @@ if (!isPublishing) {
...
+//       ' has "main" ' +
+//       'set to "' +
+//       json.main +
+//       '" in "src" but it should refer to "dist". ' +
+//       'This is an error and likely due to running hoistDependencies.js and ' +
+//       'committing or keeping the result. Please revert those changes and ' +
+//       'try again.';
+//     throw new Error(message);
+//   }
+// });
```
Comment (benjie) [inline resources/prepublish.js]: Are we going to need this again, or can we delete it?

**Comment by acao** [comment]
We are not autogenerating this unfortunately, and the git history is broken so this took 6 hours haha! We need to introduce semantic release, I have a github issue for it. Ill come back to this next chance I get in the next few days

**Comment by acao** [comment]
@benjie i made the requested changes the other night, are we good to go on this you think?

**Pull request merged**